.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
+current_version = 0.8.4
```

Comment:
oh and if we need to duplicate current version here it would ruin all point of `auto` . Makes me even more convinced that we should not use it